documentation/en/source/pages/migration/migrate-system.rst

```diff
@@ -15,6 +15,8 @@ Optionally, at step #2, you can import the scan report to create a template. Thi
 
 First, scan the system you wish to migrate by running ``scan run``. This "deep scans" the live system, reporting back the meta-data of every file and package that makes up the running workload. The following is an example of a scan of a live system:
 
+.. note:: In this example, we run a regular scan (without included overlay). If you want it, add --overlay argument to the command.
```

Comment:
The following example shows a simple scan (without overlay). If you would like the overlay, add ``--overlay`` (make sure to use ``)  argument to the command.
Somewhere (not sure here is the place) we need to explicitly describe what is the difference between a "regular" or "simple" scan vs overlay.